### PR TITLE
Feature: Support PHP 8.2

### PR DESCRIPTION
**Pull request by ghostwriter**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

- Update composer config `platform.php` to `8.0.99`
- Update composer `PHP` to `~8.0.0 || ~8.1.0 || ~8.2.0`
- Ignore PHP platform requirements via `.laminas-ci.json`
- Fix CS issues & Update TwigEnvironmentFactoryTest

### CODE REVIEW

File: .laminas-ci.json [hunk 1]
```diff
@@ -1,2 +1,5 @@
 {
-}
+    "ignore_php_platform_requirements": {
```
Comment (gsteel) [inline .laminas-ci.json:2]: Are we sure this is required? All the top-level deps support 8.2

File: src/TwigEnvironmentFactory.php [hunk 2]
```diff
@@ -229,7 +229,7 @@ private function loadRuntimeLoader($runtimeLoader, ContainerInterface $container
             throw new Exception\InvalidRuntimeLoaderException(sprintf(
                 'Twig runtime loader must be an instance of %s; "%s" given,',
                 RuntimeLoaderInterface::class,
-                is_object($runtimeLoader) ? get_class($runtimeLoader) : gettype($runtimeLoader)
+                is_object($runtimeLoader) ? $runtimeLoader::class : gettype($runtimeLoader)
```
Comment (gsteel) [inline src/TwigEnvironmentFactory.php:232]: This can be `get_debug_type` too no?

**Review by gsteel — APPROVED** [review]
Thanks @ghostwriter - LGTM, but just a couple of notes that are non-blockers for me

**Review by Ocramius — APPROVED** [review]
:ship: as-is - I think delivering 8.2 support is a priority over adjusting the noticed details :+1:

**Pull request merged**